src/py/flwr/server/run_serverapp.py

```diff
@@ -149,13 +151,15 @@ def run_server_app() -> None:
     server_app_dir = args.dir
     server_app_attr = getattr(args, "server-app")
 
-    # Initialize GrpcDriver
-    driver = GrpcDriver(
-        driver_service_address=args.superlink,
-        root_certificates=root_certificates,
-        fab_id=args.fab_id,
-        fab_version=args.fab_version,
+    # Create run
+    stub = GrpcDriverStub(
+        driver_service_address=args.superlink, root_certificates=root_certificates
     )
+    req = CreateRunRequest(fab_id=args.fab_id, fab_version=args.fab_version)
+    res = stub.create_run(req)
```

Comment:
We should have some error handling here and also check the gRPC status code

Reply:
But for now `create_run` won't raise any exception. Only `get_run` can raise an error when `run_id` can't be found.

Reply:
Any call that involves networking can raise an exception if the network is down

Reply:
Oh, I see what you mean. Yes, we should have error handling and retry mechanism here and in other driver methods. But I would like to open a separate PR to do this, i.e., making `GrpcDriver` reliable.